src/test/java/svc/data/citations/filters/CitationFilterTest.java

```diff
@@ -74,5 +82,33 @@ public void correctlyFiltersCitations(){
         
         assertThat(citationFilter.RemoveCitationsWithExpiredDates(CITATIONS).size(), is(0));
 	}
+	
+	@Test
+	public void correctlyKeepsOldWarrantCitations(){
+		Court COURT = new Court();
```

Comment:
Looks like there is some disagreement between tabs and spaces in this file. Can we get it formatted to at least be consistent one way or the other?